lib/registry.js

```diff
@@ -244,8 +248,12 @@ Registry.prototype.configureModel = function(ModelCtor, config) {
       modelName);
   }
 
+  var baseName = ModelCtor.base.modelName || 'PersistedModel';
+  BaseModel = this.getModel(baseName);
+  var model = BaseModel.extend(modelName, {}, settings);
```

Comment:
I think the correct way of handling remote-method inheritance is to extend strong-remoting SharedClass to support the concept of inheritance too. I.e. `MyCustomModel.sharedClass` should be linked to `Base.sharedClass` in such way that `MyCustomModel` inherits all methods from `Base`.